MAUI/Calendar/views.md

```diff
@@ -0,0 +1,190 @@
+
+this.Calendar.MonthView = new CalendarMonthView()
+{
+    ShowWeekNumber = true,
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![Week number in .NET MAUI Calendar.](images/views/maui-week-number.png)
+
+
+#### Week number appearance
+Week number Background and TextStyle can be customized in the month view. Background color can be changed by using the [Background] property and the textStyle can be changed by using the [TextStyle] property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar"  View="Month">
+            <Calendar:SfCalendar.MonthView>
+                <Calendar:CalendarMonthView ShowWeekNumber="True">
+                    <Calendar:CalendarMonthView.WeekNumberStyle>
+                        <Calendar:CalendarWeekNumberStyle Background="DeepSkyBlue">
+                            <Calendar:CalendarWeekNumberStyle.TextStyle>
+                                <Calendar:CalendarTextStyle TextColor="White" FontSize="12" />
+                            </Calendar:CalendarWeekNumberStyle.TextStyle>
+                        </Calendar:CalendarWeekNumberStyle>
+                    </Calendar:CalendarMonthView.WeekNumberStyle>
+                </Calendar:CalendarMonthView>
+            </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView = new CalendarMonthView()
+{
+    ShowWeekNumber = true,
+};
+
+CalendarTextStyle textStyle = new CalendarTextStyle()
+{
+    TextColor = Colors.Black,
+    FontSize = 12,
+};
+
+this.Calendar.MonthView.WeekNumberStyle = new CalendarWeekNumberStyle()
+{
+    Background = Colors.DeepSkyBlue,
+    TextStyle = textStyle,
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![week-number-style-in-maui-calendar](images/views/week-number-style-in-maui-calendar.png)
+
+
+## Year view
+The Year view displays the current year's month. A calendar year is a one-year period that begins on January 1 and ends on December 31. By default, initially displays the current years month and the current month is highlighted by a separate color different from the rest of the month color in `Year` view. Can easily navigate to the desired month dates from the year view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Year">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Year;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Year view in .NET MAUI Calendar.](images/views/maui-year-view.png)
+
+## Decade view
+The Decade view displays the period of ten years and some years of next view. By default, initially displays the current year view and the current year is highlighted by a separate color different from the rest of the years color in `Decade` view. Can easily navigate to the desired year in the Year view from the decade view.
```

Comment:
The Decade view shows the period of ten years and some years ahead. By default, it displays the current year view, with the current year highlighted in a different color than the other years in the 'Decade' view. From the decade view, you can easily navigate to the desired year in the Year view.

Reply:
Content changed for the decade view.